### PR TITLE
[bitnami/cert-manager] Add VIB tests

### PR DESCRIPTION
**Pull request by mdhont**
### Description of the change

The main objective of this PR is to publish our Bitnami cert-manager container using VMware Image Builder. In order to do that, several changes are included:

- Increasing the existing test coverage of the asset by adding Goss tests.
- Update verify and publish VIB pipeline's definitions.

### Benefits

- Ensuring higher quality of the container catalog.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Other

Successful run:
https://github.com/bitnami/containers/actions/runs/5001002980/jobs/8959127187

### CODE REVIEW

File: .vib/cert-manager/goss/cert-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+command:
+  # There are no syntax error in the configuration
```
Comment (migruiz4) [inline .vib/cert-manager/goss/cert-manager.yaml:2]: What do you mean by this comment?

**Comment by migruiz4** [comment]
The PR looks good to me, but could you please take a look at my comment before I approve it?

File: .vib/cert-manager/goss/cert-manager.yaml (hunk 1 above)
Comment (mdhont) [reply .vib/cert-manager/goss/cert-manager.yaml]: @migruiz4, a copy paste error. Fixed it now

File: .vib/cert-manager/goss/cert-manager.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  check-cert-manager:
+    exec: /opt/bitnami/cert-manager/bin/cert-manager -h
```
[the remark this answers is not in the document]
Comment (mdhont) [reply .vib/cert-manager/goss/cert-manager.yaml:3]: @migruiz4, isn't that done in the binaries section? In any case, using the full path would check the path and the binary, right?

[the remark this answers is not in the document]
Comment (migruiz4) [reply .vib/cert-manager/goss/cert-manager.yaml:3]: I think tests should abstract the `/opt/bitnami/` path, that is why we add the `root_dir` in `vars.yaml`.

If you see tests that we have implemented so far, none of them used the full path

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Please take a look at my pending comment: https://github.com/bitnami/containers/pull/34256/files#r1198978792

File: .vib/cert-manager/goss/cert-manager.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mdhont) [reply .vib/cert-manager/goss/cert-manager.yaml:3]: Ok, I will change it. I've based using the full path on this comment:

https://github.com/bitnami/containers/pull/32552#discussion_r1186174744

**Review by migruiz4 — APPROVED** [review]
Looks good to me!

**Pull request merged**